pkg/kubelet/network/kubenet/kubenet_linux.go

```diff
@@ -494,6 +484,26 @@ func (plugin *kubenetNetworkPlugin) Status() error {
 	return nil
 }
 
+// getContainerID returns the containerID that will be used to retrieve the pod IP.
+// For Docker, the container ID is the pod infra container's ID.
+// For rkt, the container ID is the API pod's UID.
+// TODO(yifan): Rename to getPodID.
```

Comment:
@freehan There are two IDs associated with a pod, the pod's API UID, and the runtime pod's ID which is allocated by the runtime. We probably want to use the API UID because the other one is not available before the pod is created.